pyproject.toml

```diff
@@ -1,3 +1,90 @@
 [build-system]
-requires = ["setuptools>=41.2", "setuptools_scm", "wheel"]
-build-backend = "setuptools.build_meta"
+requires = ["hatchling", "hatch-requirements-txt", "hatch-vcs"]
+build-backend = "hatchling.build"
+
+[project]
+name = "conda_lock"
+description = "Lockfiles for conda"
+authors = [
+    { "name" = "Marius van Niekerk", email = "marius.v.niekerk@gmail.com" },
+]
+readme = "README.md"
+license = "MIT"
+requires-python = ">=3.6"
```

Comment:
fyi Hatchling is 3.7+ so builder job can't use that

Reply:
but hatch can still build for older pythons right?  its just metadata

Reply:
yes

Reply:
Ok, so we might need to adjust the workflows so that we build with 3.7+ before testing with 3.6.

But @mariusvniekerk, is there any benefit to supporting 3.6? Conda-forge stopped 3.6 builds some time ago. Plus, one could always `condax install` conda-lock to have it run in an isolated environment with modern Python.